src/status_im2/contexts/chat/lightbox/zoomable_image/utils.cljs

```diff
@@ -102,6 +105,27 @@
   (when (and (= zoom-out-signal index) (> scale c/min-scale))
     (rescale c/min-scale true)))
 
+(defn toggle-opacity
```

Comment:
I see you haven't changed `toggle-opacity`, but since it's in the PR I'll comment anyways, but feel free to ignore.

I think all timers should be cancelled on the use-effect's final return function.